GliaWidgets/ViewController/Survey/SurveyViewController.View.swift

```diff
@@ -162,3 +150,25 @@ extension NSAttributedString {
         return mutableString
     }
 }
+
+extension UIButton {
+    func update(with style: Theme.Button) {
+        style.background.map { backgroundColor = UIColor(hex: $0) }
```

Comment:
While it's convenient to use map like that, I think it goes against the purpose of map operation (transforming values).
Is it possible to use it like this?
```
backgroundColor = style.background.map(UIColor.init(hex:))
```
Does it make sense?

Reply:
![image](https://user-images.githubusercontent.com/28916962/172229914-c97ad2ee-9f74-4f2c-8c22-184827cee037.png)
I thought the same, but for some reason, Xcode throws error...